### ISSUE TITLE
Skill management in class (adding, removing - editing already done)

### ISSUE THREAD

**Issue by adamskibicki** [adamskibicki/levelling-assistant#37]
requires: https://github.com/adamskibicki/Progress.API/issues/18

**Comment by adamskibicki** [comment]
error when adding new skill variable - it is not in redux state but is in this skill state
it happens because useCalculateValueOfIncreasedVariable in SkillVariableComponent uses data from redux state
1. add this data instantly to state on skill variable edit modal accept - correct - redux state decides what is on page
 but what about when i add a skill variable but cancel adding new skill - what now?

2. make this data independent of redux state (it stays as component state) - need making all hooks independent of redux state and forces to pass all
 data from components - components will handle everything here - i think this is a correct way to handle that
 or avoid using hooks dependant on redux state or give them ability to handle skillvariable calculations for variables that are not in state

After consideration solution 2 will be implemented
